consensus/byzantine_test.go

```diff
@@ -60,6 +59,8 @@ func TestByzantinePrevoteEquivocation(t *testing.T) {
 		})
 		state, _ := stateStore.LoadFromDBOrGenesisDoc(genDoc)
 		thisConfig := ResetConfig(fmt.Sprintf("%s_%d", testName, i))
+		nodeKey, err := p2p.LoadOrGenNodeKey(thisConfig.NodeKeyFile())
+		require.NoError(t, err)
```

Comment:
this might have been a silly merge regression, but whenever we use these keys in tests and they aren't otherwise used we need to use this method and the method to create the entire node key path cc @rach-id

Reply:
thanks man